ee/engine-server-ee/build.sh

```diff
@@ -6,4 +6,4 @@ yarn install
 node ee/engine-server-ee/esbuild.js
 cd server
 echo "{}" > package.json
-yarn add bcrypt heapdump
+yarn add bcryptjs heapdump
```

Comment:
not needed anymore. it was here because bcrypt was external dependency in `scripts/esbuild/esbuild.js`

Reply:
removed